docs/meshcloud.metadata-tags.md

```diff
@@ -27,6 +27,11 @@ Administrators can configure tags so that only [Partner users](./administration.
 
 The [administration documentation](./administration.index.md) provides further details on how partner users can edit restricted tags.
 
+## Immutable tags
+
+Administrators can configure immutable tags. This means that we allow the selection of specific tag values only during the creation process [meshCustomers](meshcloud.customer.md), [meshProjects](meshcloud.project.md), [meshLandingZones](meshcloud.landing-zones.md) and [meshPaymentMethods](./meshcloud.payment-methods.md). Afterwards it is no longer possible to edit the selected values. Also administrators cannot change immutable tag values after creation.
```

Comment:
Immutable tags can also be a string like coriaId  or I think from the term we can use set instead select to cover the different data types
```suggestion
Administrators can configure immutable tags. This means that we allow to set specific tag values only during the creation process [meshCustomers](meshcloud.customer.md), [meshProjects](meshcloud.project.md), [meshLandingZones](meshcloud.landing-zones.md) and [meshPaymentMethods](./meshcloud.payment-methods.md). Afterwards it is no longer possible to edit the values. Also administrators cannot change immutable tag values after creation.
```

Reply:
Please take a look into the PR https://github.com/meshcloud/meshfed-release/pull/2377